libcloudforensics/providers/aws/internal/account.py

```diff
@@ -104,13 +104,13 @@ def ListInstances(self, region=None, filters=None, show_terminated=False):
       region (str): Optional. The region from which to list instances.
           If none provided, the default_region associated to the AWSAccount
           object will be used.
-      filters (list(dict)): Optional. Filters for the query.
+      filters (list[dict[str, str|list[str]]]): Optional. Filters for the query.
```

Comment:
Same, this sounds like it could possible break easily

Reply:
According to boto3's doc, filters are just `dict` structures which keys (of type `str`) map directly to another `str` or to a list of `str`. How would you like to have this kind of type declared?